core/tests/Test.Microsoft.Identity.Core.UIAutomation/CoreMobileTestHelper.cs

```diff
@@ -61,6 +58,32 @@ public void PerformSignInFlowWithoutUI(ITestController controller)
             controller.Tap(CoreUiTestConstants.AcquireTokenID);
         }
 
+        public void PerformB2CLocalAccountSignInFlow(ITestController controller, LabUser user)
```

Comment:
With ADAL not supporting B2C, this should probably be in the MSALMobileTestHelper as ADAL should never use it. However, I don't see much harm in leaving this here.

Reply:
definitely. good idea. done.

---
In reply to: [233220687](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1403#discussion_r233220687) [](ancestors = 233220687)